idea_town/frontend/static-src/app/views/#experiment-page.js#

```diff
@@ -0,0 +1,249 @@
+import app from 'ampersand-app';
```

Comment:
Is this file meant to be here? `#experiment-page.js#` seems like maybe a backup filename or something.